liquibase-core/src/main/java/liquibase/change/core/RawSQLChange.java

```diff
@@ -10,29 +10,11 @@
 import liquibase.util.StringUtil;
 
 /**
- * Allows execution of arbitrary SQL.  This change can be used when existing changes are either don't exist,
+ * Allows execution of arbitrary SQL. This change can be used when existing changes are either don't exist,
  * are not flexible enough, or buggy. 
  */
-@DatabaseChange(name="sql",
```

Comment:
This is another chunk of information that should be available, although I agree it should not be in the JavaDocs directly. Should I enter PD tickets for these sorts of things, @adrian-velonis1 ?

Reply:
I think this info should be (mostly? Completely?) covered by the docs page on the [<sql>](https://docs.liquibase.com/change-types/sql.html) changetype, where it is also formatted better. If there is anything that needs clarification on that page, we can definitely make adjustments.

If you find anything that shouldn't be in Javadocs but should be in the user-facing docs somewhere, definitely make a PD ticket and tag one of us!